site/components/footer/index.js

```diff
@@ -20,31 +21,43 @@ import mapSVG from './SVG/map.svg';
 
 const cx = classnames.bind(styles);
 
+const trackAnalytics = title => () => ReactGA.event({
+  category: 'FooterNavigation',
+  action: title,
+  label: 'click',
+});
+
 const Footer = () => (
   <footer role="contentinfo" className={styles.footer}>
     <div className={styles.footerContainer}>
-      <InlineSVG role="presentation" src={mapSVG} className={styles.footerMap} />
+      <InlineSVG
+        role="presentation"
+        src={mapSVG}
+        className={styles.footerMap}
+      />
       <div className={styles.footerSections}>
         <nav role="navigation" className={cx('section', 'footerLinks')}>
           <ul className={styles.nav}>
+            {/* eslint-disable jsx-a11y/no-static-element-interactions */}
             <li>
-              <Link to="homePage">Home</Link>
+              <Link to="homePage"><span onClick={trackAnalytics('Home')}>Home</span></Link>
             </li>
             <li>
-              <Link to="aboutUsPage">About us</Link>
+              <Link to="aboutUsPage"><span onClick={trackAnalytics('About us')}>About us</span></Link>
             </li>
             <li>
-              <Link to="whatWeDoPage">What we do</Link>
+              <Link to="whatWeDoPage"><span onClick={trackAnalytics('What we do')}>What we do</span></Link>
             </li>
             <li>
-              <a href="/blog/">Blog</a>
+              <a href="/blog/"><span onClick={trackAnalytics('Blog')}>Blog</span></a>
```

Comment:
Tested with Google Tag analyser for chrome. Might want to double check because blog isn't served locally. So it may only be doing half a redirect then crashing. @gregdorward something to watch out for when testing. I will add to the test plan